text-utilities/utilities.html

```diff
@@ -0,0 +1,98 @@
+                </li>
+                <li class="compo-scroll-list fs-xl">
+                    <a href="/buttons/buttons.html" class="style-anchor-list">Buttons</a>
+                </li>
+                <li class="compo-scroll-list fs-xl">
+                    <a href="/image-component/image.html" class="style-anchor-list">Image</a>
+                </li>
+                <li class="compo-scroll-list fs-xl">
+                    <a href="/input/input.html" class="style-anchor-list">Input</a>
+                </li>
+                <li class="compo-scroll-list fs-xl"><a href="/text-utilities/utilities.html"
+                        class="style-anchor-list">Utilities</a></li>
+                <li class="compo-scroll-list fs-xl"><a href="/list/list.html" class="style-anchor-list">Lists</a> </li>
+                <li class="compo-scroll-list fs-xl"><a href="/navigation/navigation.html"
+                        class="style-anchor-list">Navigation</a></li>
+                <li class="compo-scroll-list fs-xl"><a href="/rating/rating.html" class="style-anchor-list">Ratings</a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="component-data-display-container">
+            <div class="text-heading">
+                <h1>TEXT-UTILITY</h1>
+            </div>
+
+            <div class="div-centered">
+                <h2>Types of Headings</h2>
+            </div>
+
+
+            <div class="types-of-headings mrg-lrg div-centered">
+                <div id="small-heading">Small Heading</div>
+                <div id="medium-heading">Medium Heading</div>
+                <div id="large-heading">Large Heading</div>
+            </div>
+
+            <div class="div-centered mrg-med">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522types-of-headings%2522%253E%250A%2520%2520%253Cdiv%2520id%253D%2522small-heading%2522%253ESmall%2520Heading%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520id%253D%2522medium-heading%2522%253EMedium%2520Heading%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520id%253D%2522large-heading%2522%253ELarge%2520Heading%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                    style="width: 546px; height: 279px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+            <div class="div-centered">
+                <h2>Types of Texts</h2>
+            </div>
+
+            <div class="type-text-container div-centered">
+                <p class="para-type">This is bold text.</p>
+                <p class="para-type">This is center text.</p>
+                <p class="para-type">This is gray text.</p>
+            </div>
```

Comment:
Suggestion: I guess you can make separate class for center text and show is in a separate div to be more precise.